### DISCUSSION TITLE
Given a short address, convert it or obtain a long one (f0 -> f4)

### DISCUSSION THREAD

**Discussion by GildedHonour** [filecoin-project/lotus#11096] [General Q&A]
Given a short address f0, can it be converted into a long one -- f4? Preferably, locally without any request. Otherwise, via a RPC request to a node. Otherwise,via a third-party gateway API.

@TippyFlitsUK Who knows better whether or not my question has been answered: you or me?

**Comment by TippyFlitsUK** [comment]
Please stop spamming this channel with duplicate questions @GildedHonour!!

You have an open discussion which is still ongoing which addresses your question!

https://github.com/filecoin-project/lotus/discussions/10898#discussioncomment-6502421
